src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <nav :class="breadcrumbClasses">
+        <ul>
+            <template v-for="(item, index) in items">
+                <li
+                    v-if="item"
+                    :key="item"
+                    :class="{ 'is-active': index === items.length - 1 }">
+                    <router-link
+                        :to="item.path"
+                        tag="a">
+                        {{ item.label }}
+                    </router-link>
+                </li>
+            </template>
+        </ul>
+    </nav>
+</template>
+
+<script>
+import config from '../../utils/config'
+
+export default {
+
+    name: 'BBreadcrumb',
+
+    props: {
+        items: Array,
+        align: {
+            type: String,
+            default: () => {
+                return config.defaultBreadcrumbAlign
+            }
+        },
+        separator: {
+            type: String,
+            default: () => {
+                return config.defaultBreadcrumbSeparator
+            }
+        },
+        size: {
+            type: String,
+            default: () => {
+                return config.defaultBreadcrumbSize
+            }
+        }
+    },
+
+    data() {
+        return {
+            bulmaKnownAlignments: ['is-left', 'is-centered', 'is-right'],
```

Comment:
The following data doesn't make sense, please follow the same structure of other components.
Users already know that size is "is-small", etc (same for alignments), anyway, you can add them in the API doc of component